src/PockyBot.NET/Persistence/Repositories/PockyUserRepository.cs

```diff
@@ -18,24 +18,23 @@ public PockyUserRepository(DatabaseContext context)
         public PockyUser GetUser(string userId)
         {
             return _context.PockyUsers
-                .Where(x => x.UserId == userId)
                 .Include(x => x.PegsGiven)
                     .ThenInclude(x => x.Receiver)
                 .Include(x => x.PegsReceived)
                 .Include(x => x.Location)
                 .Include(x => x.Roles)
-                .SingleOrDefault();
+                .SingleOrDefault(x => x.UserId == userId);
```

Comment:
what does this change do?

Reply:
It moves the logic from the where into the singleordefault.

I changed it to use a where for performance reasons, so I think we should move it back. The reason is you want to do the filtering before you do any table joins, as table joins are expensive

Reply:
Honestly I made these changes when I was trying to diagnose the issue, no problems with changing them back